ci/windows_ci_steps.sh

```diff
@@ -16,10 +16,18 @@ df -h
 # TODO(dio): Put in windows/.bazelrc.
 export PATH="/c/Program Files (x86)/Windows Kits/10/bin/10.0.17763.0/x64":$PATH
 
-BAZEL_STARTUP_OPTIONS="--bazelrc=windows/.bazelrc"
-BAZEL_BUILD_OPTIONS="--show_task_finish --verbose_failures \
+# Set up TMPDIR so bash and non-bash can access
+# e.g. TMPDIR=/d/tmp, make a link from /d/d to /d so both bash and Windows programs resolve the
+# same path
+drive="$(readlink -f $TMPDIR | cut -d '/' -f2)"
+/c/windows/system32/cmd.exe "/c mklink /d $drive:\\$drive $drive:\\"
```

Comment:
[This issue](https://github.com/bazelbuild/rules_foreign_cc/issues/334) (rules_foreign_cc spits out temp files using `mktemp` via bash which puts things into paths like `/tmp` which only make sense to bash, other tools that are running directly on Windows don't fully get these paths) causes us to have a hacky workaround here (and otherwise). We had problems making symlinks on the Azure CI from the Git bash `/tmp` dir (actually `C:\Program Files\Git\tmp`) to a directory that would properly resolve `/tmp` as a Windows path (e.g. `D:\tmp` or `C:\tmp`). Instead, we coerce `mktemp` to put files in the directory pointed at by the `TMPDIR` env variable and link that to the working drive. So as in the comment example a path that looks like `/d/tmp/foo` returned by `mktemp` from bash would be resolvable to `D:\tmp\foo` b/c of the symlink. Let us know if you need any more clarification.

Reply:
I think put the link to that rules_foreign_cc issue in the comment will be helpful.

Reply:
should be updated!